Niak 102

jakubowiczish opened the pull request.

jakubowiczish committed:
[NIAK-102] Add basis of returning ExamTerms which are not deleted.

jakubowiczish committed:
[NIAK-102] Add set deleted endpoint for exam term

jakubowiczish committed:
[NIAK-102] Add endpoint for updating exam term

jakubowiczish committed:
[NIAK-102] Update postman collection and environment

jfoder commented on rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/model/database/ExamTerm.java:
```diff
@@ -45,6 +49,10 @@
 
+    @NonNull
+    @ColumnDefault("false")
+    private Boolean isDeleted;
```
Chyba lepiej nazywać booleany bez _is_, bo później mogą być problemy przy parsowaniu np. Jacksonem:
https://stackoverflow.com/questions/32270422/jackson-renames-primitive-boolean-field-by-removing-is

jfoder commented on rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/model/rest/examterm/UpdateExamTermRequest.java:
```diff
@@ -0,0 +1,42 @@
+    private Long id;
+
+    @NonNull
+    private Boolean isDeleted;
```
Jak wyżej

jakubowiczish committed:
[NIAK-102] Change name of variable deleted for exam term

jakubowiczish committed:
[NIAK-102] Update

jakubowiczish committed:
[NIAK-102] Merge branch 'develop' into NIAK-102

# Conflicts:
#	rezerwacje-app/src/test/resources/postman/reservation-system.postman_collection.json
#	rezerwacje-app/src/test/resources/postman/reservation-system.postman_environment.json
